packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx

```diff
@@ -17,19 +17,19 @@ Here is a list of things still to address on this page:
 There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in video components ready for use.
 In addition, it allows for styling and custom theming options with custom CSS.
 
-For custom use cases it is also possible to only use the low-level client to provide completely custom UI to fully tailor the SDK towards your needs.
+For custom use cases it is also possible to create your own components to fully tailor the SDK towards your needs. Please note that we suggest not to override a few, core components as those contain complex, low-level logic related to performance, the list of these components can be found in the [UI Components overview](../03-ui/01-overview.mdx).
 
 This page guides you through all of them with increasing levels of customization.
 
-### Using pre-built components
+## Using pre-built components
 
 The SDK comes with a lot of pre-built components for the different elements of the UI.
 
 :::tip
 For a list of all available UI components head over to the [Components](../../ui/01-ui-overview) section.
```

Comment:
```suggestion
For a list of all available UI components head over to the [Components](../../03-ui/01-overview) section.
```

Reply:
The link works without the changes, I think there are two link formats: one is URL based and the other is file based